scripts/update_specs_services_from_ssm.py

```diff
@@ -0,0 +1,285 @@
+}
+
+service_map = {
+    'acm': ['AWS::CertificateManager::'],
+    'apigateway': ['AWS::ApiGateway::', 'AWS::ApiGatewayV2::'],
+    'application-autoscaling': ['AWS::ApplicationAutoScaling::'],
+    'appstream': ['AWS::AppStream::'],
+    'appsync': ['AWS::AppSync::'],
+    'athena': ['AWS::Athena::'],
+    'autoscaling': ['AWS::AutoScaling::'],
+    'batch': ['AWS::Batch::'],
+    'budgets': ['AWS::Budgets::'],
+    'cloud9': ['AWS::Cloud9::'],
+    'cloudfront': ['AWS::CloudFront::'],
+    'cloudtrail': ['AWS::CloudTrail::'],
+    'cloudwatch': ['AWS::CloudWatch::'],
+    'codebuild': ['AWS::CodeBuild::'],
+    'codecommit': ['AWS::CodeCommit::'],
+    'codedeploy': ['AWS::CodeDeploy::'],
+    'codepipeline': ['AWS::CodePipeline::'],
+    'cognito-identity': ['AWS::Cognito::'],
+    'config': ['AWS::Config::'],
+    'datapipeline': ['AWS::DataPipeline::'],
+    'dax': ['AWS::DAX::'],
+    'dms': ['AWS::DMS::'],
+    'docdb': ['AWS::DocDB::'],
+    'ds': ['AWS::DirectoryService::'],
+    'dynamodb': ['AWS::DynamoDB::'],
+    'ec2': ['AWS::EC2::'],
+    'ecr': ['AWS::ECR::'],
+    'ecs': ['AWS::ECS::'],
+    'efs': ['AWS::EFS::'],
+    'eks': ['AWS::EKS::'],
+    'elasticache': ['AWS::ElastiCache::'],
+    'elasticbeanstalk': ['AWS::ElasticBeanstalk::'],
+    'elb': ['AWS::ElasticLoadBalancing::', 'AWS::ElasticLoadBalancingV2::'],
+    'emr': ['AWS::EMR::'],
+    'es': ['AWS::Elasticsearch::'],
+    'events': ['AWS::Events::'],
+    'firehose': ['AWS::KinesisFirehose::'],
+    'fsx': ['AWS::FSx::'],
+    'gamelift': ['AWS::GameLift::'],
+    'glue': ['AWS::Glue::'],
+    'greengrass': ['AWS::Greengrass::'],
+    'guardduty': ['AWS::GuardDuty::'],
+    'inspector': ['AWS::Inspector::'],
+    'iot': ['AWS::IoT::'],
+    'iot1click-projects': ['AWS::IoT1Click::'],
+    'iotanalytics': ['AWS::IoTAnalytics::'],
+    'kinesis': ['AWS::Kinesis::'],
+    'kinesisanalytics': ['AWS::KinesisAnalytics::', 'AWS::KinesisAnalyticsV2::'],
+    'kms': ['AWS::KMS::'],
+    'lambda': ['AWS::Lambda::'],
+    'logs': ['AWS::Logs::'],
+    'mq': ['AWS::AmazonMQ::'],
+    'neptune': ['AWS::Neptune::'],
+    'opsworks': ['AWS::OpsWorks::'],
+    'opsworkscm': ['AWS::OpsWorksCM::'],
+    'ram': ['AWS::RAM::'],
+    'rds': ['AWS::RDS::'],
+    'redshift': ['AWS::Redshift::'],
+    'robomaker': ['AWS::RoboMaker::'],
+    'route53': ['AWS::Route53::'],
+    'route53resolver': ['AWS::Route53Resolver::ResolverRule', 'AWS::Route53Resolver::ResolverEndpoint'],
+    's3': ['AWS::S3::'],
+    'sagemaker': ['AWS::SageMaker::'],
+    'sdb': ['AWS::SDB::'],
+    'secretsmanager': ['AWS::SecretsManager::'],
+    'servicecatalog': ['AWS::ServiceCatalog::'],
+    'servicediscovery': ['AWS::ServiceDiscovery::'],
+    'ses': ['AWS::SES::'],
+    'sns': ['AWS::SNS::'],
+    'sqs': ['AWS::SQS::'],
+    'ssm': ['AWS::SSM::'],
+    'stepfunctions': ['AWS::StepFunctions::'],
+    'waf-regional': ['AWS::WAFRegional::'],
+    'workspaces': ['AWS::WorkSpaces::'],
+}
```

Comment:
thinking it might be easier to just maintain the services that aren't case-insensitive matches and find a way to automatically map all the services that are case-insensitive matches:
```python
service_map = {
    'acm': ['AWS::CertificateManager::'],
    'apigateway': ['AWS::ApiGateway::', 'AWS::ApiGatewayV2::'],
    # 'application-autoscaling': ['AWS::ApplicationAutoScaling::'], ## remove because SSM endpoints aren't correct
    'cognito-identity': ['AWS::Cognito::'],
    'ds': ['AWS::DirectoryService::'],
    'elb': ['AWS::ElasticLoadBalancing::', 'AWS::ElasticLoadBalancingV2::'],
    'es': ['AWS::Elasticsearch::'],
    'firehose': ['AWS::KinesisFirehose::'],
    'iot1click-projects': ['AWS::IoT1Click::'],
    'kinesisanalytics': ['AWS::KinesisAnalytics::', 'AWS::KinesisAnalyticsV2::'],
    'mq': ['AWS::AmazonMQ::'],
    'route53resolver': ['AWS::Route53Resolver::ResolverRule', 'AWS::Route53Resolver::ResolverEndpoint'],
    'waf-regional': ['AWS::WAFRegional::'],
}
```